### PR TITLE
VeChain Grant Proposal -- VeTools (VeChain VSCode Toolkit)

### PR DESCRIPTION
**Pull request by hazelcaus**
# Grant Application Checklist

- [x] The [`applications/application-template.md`](../applications/application-template.md) or [`applications/research-template.md`](../applications/research-template.md) has been copied, renamed ( "project_name.md") and updated.
- [x] An Ethereum address for the payment of the milestones is provided inside the application.  
- [x] The total funding amount of the project is up to USD $30k for every grant applications.
- [x] The initial PR contains only one commit (squash if needed before submitting your PR).

### CODE REVIEW

File: applications/vetools_vscode.md [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+- https://github.com/hazelcaus
+
+### Team LinkedIn Profiles
+
+- https://www.linkedin.com/in/magnus-ekstr%C3%B6m-46b993278/
+
+
+## Development Roadmap 
+
+This section should break out the development roadmap into a number of milestones. Since the milestones will appear in the grant contract, it helps to describe the functionality we should expect, plus how we can check that such functionality exists in the product. Whenever milestones are delivered, we refer to the contract to ensure that everything has been delivered as expected.
```
Comment (libotony) [inline applications/vetools_vscode.md:69]: This section should be deleted.

File: applications/vetools_vscode.md [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+Below we provide an <b>example roadmap</b>. For each milestone:
+
+- Please indicate the milestone duration, workload in terms of full-time equivalent (FTE) and cost. 
+- Please be sure to include a specification of the software. The level of details must be high enough so that we are able to verify that the software meets the specification.
+- Please note that we require documentation (e.g. tutorials, API specifications, architecture details) in each milestone. This ensures that the code can be widely used by the community.
+- Please provide a test suite, comprising unit and integration tests, along with a guide on how to run these.
+- Please commit to providing dockerfiles for the delivery of your project.
+
+
+### Example Roadmap for a dApp Application
```
Comment (libotony) [inline applications/vetools_vscode.md:80]:
```suggestion
```

This section should be removed.

**Comment by hazelcaus** [comment]
Hey @libotony -- thanks. Just removed that section!

**Review by NecoSherry — APPROVED** [review]

**Review by libotony — DISMISSED** [review]

**Comment by NecoSherry** [comment]
Congratulations! Your application has been accepted by the committee!

We would like to inform you that we have recently updated our payment currency to USDC (ERC 20). Therefore, we kindly ask you to confirm that your payment address is capable of receiving USDC payments. To ensure a smooth process, we will make a small test payment after your confirmation before the first payout.

Regarding the milestones, we kindly ask for your assistance in confirming the cost for milestone 3 and the total cost. This information is crucial for us to proceed with the grant disbursement accurately.

In order to proceed with further payments, we need to contact you by email. Please send an email to [grant@vechain.org](mailto:grant@vechain.org) with your project name, PR link and payment address (same as in the PR).

**Comment by hazelcaus** [comment]
Hi @NecoSherry! We're so excited! 

- Can confirm my payment address can handle USDC payments
- Just updated the milestones and made it clearer. There are 3 milestones of $10k each -- so, total is $30k. I also realize that this just botched your previous reviews, so kindly requesting you to review the latest changes @NecoSherry @libotony . Thanks
- Will gladly send the email -- but do I wait for this PR to be reviewed again in light of my changes above ^

**Pull request merged**